src/setupProxy.js

```diff
@@ -22,6 +22,9 @@ module.exports = (app) => {
   );
 
   // @LOCALHOST
-  // app.use('/api', createProxyMiddleware({ target: 'http://localhost:2477/', changeOrigin: true }));
-  // app.use('/token', createProxyMiddleware({ target: 'http://localhost:2477/', changeOrigin: true }));
+  // app.use('/api', createProxyMiddleware({ target: 'http://localhost:1995/', changeOrigin: true }));
+  // app.use(
+  //   '/token',
+  //   createProxyMiddleware({ target: 'http://localhost:1995/', changeOrigin: true }),
+  // );
```

Comment:
Please undo this like this.

// app.use('/api', createProxyMiddleware({ target: 'http://localhost:2477/', changeOrigin: true }));
// app.use('/token', createProxyMiddleware({ target: 'http://localhost:2477/', changeOrigin: true }));

Reply:
Yes, no changes to setupProxy

Reply:
Done!!! @iamphg97